### ISSUE TITLE
Prep our starting recommendations for scans 

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#698]
For: 
* accessibility 
* mobile
* SEO 
* customer satisfaction

**Comment by gbinal** [comment]
## a11y

One list: 

- Low contrast text - might already have
- Missing alt text  - might already have 
- Empty links
- Missing labels - might already have
- Empty buttons
- Missing language

Another (lighthouse): 
* https://developer.chrome.com/docs/lighthouse/accessibility/scoring

**Comment by gbinal** [comment]
done
